### PR TITLE
Copy zipkin functionality from java-spring-cloud

### PR DESCRIPTION
**Pull request by geoand**

### CODE REVIEW

File: .travis.yml [hunk 1]
```diff
@@ -0,0 +1,45 @@
+sudo: required
+dist: trusty
+
+language: java
+jdk:
+  - oraclejdk8
+
+services:
```
Comment (pavolloffay) [inline .travis.yml]: I think it's not needed

File: README.md [hunk 2]
```diff
@@ -0,0 +1,97 @@
+[![Build Status][ci-img]][ci] [![Released Version][maven-img]][maven]
```
Comment (pavolloffay) [inline README.md:1]: Links for these badges are missing

File: README.md [hunk 3]
```diff
@@ -0,0 +1,97 @@
...
+
```
Comment (pavolloffay) [inline README.md:2]: Top level title is missing + description

**Review by pavolloffay — APPROVED** [review]
Just some minor comments

File: .travis.yml (hunk 1 above)
Comment (geoand) [reply .travis.yml]: That is correct, I will fix it

File: README.md (hunk 2 above)
Comment (geoand) [reply README.md:1]: Done

File: README.md (hunk 3 above)
Comment (geoand) [reply README.md:2]: Done

**Pull request merged**